i18n/en.json

```diff
@@ -8,5 +8,9 @@
 	"wikibasereconcileedit-desc": "This is the ON OR Prototype extension",
 	"wikibasereconcileedit-editendpoint-invalid-reconcile-json": "Invalid reconcile JSON supplied",
 	"wikibasereconcileedit-editendpoint-unsupported-reconcile-version": "The only supported reconciliation {{PLURAL:$2|version is|versions are}} $1",
-	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123"
+	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123",
+	"wikibasereconcileedit-editendpoint-invalid-property-type-must-be-url": "urlReconcile property must be of type url",
+	"wikibasereconcileedit-editendpoint-unspecified-entity-input-version": "entity input version must be specified",
```

Comment:
I would use the opportunity in this pull request to at least begin this and the next message with an uppercase character (“Entity”, “Unknown”), to have a little bit of consistency in the messages.

Reply:
Done

Reply:
Done.